instrumentation/opentelemetry-instrumentation-jinja2/src/opentelemetry/instrumentation/jinja2/__init__.py

```diff
@@ -52,7 +52,7 @@
 from opentelemetry.instrumentation.jinja2.version import __version__
 from opentelemetry.instrumentation.utils import unwrap
 from opentelemetry.trace import SpanKind, get_tracer
-from opentelemetry.trace.status import Status, StatusCanonicalCode
+from opentelemetry.trace.status import Status, StatusCode
```

Comment:
It doesnt look like this is used anywhere.

Reply:
Opened https://github.com/open-telemetry/opentelemetry-python-contrib/issues/124 to track removing the import